### ISSUE TITLE
Threshold for intent detection

### ISSUE THREAD

**Issue by Aries-Spring** [axa-group/nlp.js#1285]
Is there any way to change threshold/sensitivity for intent detection?

Currently I trained a model on these messages:

```
// "message.delete" intent
Delete last message by %user%
Delete last %count% messages by %user%
Delete last message in %channel%
Delete last message from %user% in %channel%
Delete last %count% messages from %user% in %channel%
```

Now any prompt containing the word "message" is marked with "message.delete" intent

Is there any way to change sensitivity so it doesn't mark intents that aggressively?

**Comment by alacarte42** [comment]
yeah... this is pretty crucial. i just poked around for an hour trying to find anything, NER has a threshold setting, but it appears no way to set it in Nlp manager... which makes this library useless for my application of passing the message on to a LLM when intent is not recognized.

**Comment by alacarte42** [comment]
[SOLUTION]
this pattern did the trick to register action on intent with fine-tuned score, or define a single variable:
`await nlp.process('en', message).then(async result => {
  const intent = result.intent;
  const score = result.score;
 if (intent='greetings.hello' && score>=0.99) {
  } else if (intent='greetings.bye' && score>=0.99) {
  } else if (intent='greetings.name' && score>=0.99) {
  } else if (intent='weather' && score>=0.99) {
  } else { //default intent:none
  }`
